frontend/src/config/i18n/de/guidedTour.json

```
@@ -0,0 +1,28 @@
+{
+    "next": "Weiter",
+    "finish": "Abschließen",
+    "back": "Zurück",
+    "mapEditor": {
+        "welcome_title": "Willkommen im Map Editor",
+        "toolboxIntro_title": "Toolbox",
+        "layersIntro_title": "Ebenen",
+        "timelineIntro_title": "Zeitleiste",
+        "baseLayer_title": "Basis Ebene",
+        "plantsLayer_title": "Pflanzen Ebene",

```

```suggestion
        "plantsLayer_title": "Pflanzen Ebene",
```suggestion
        "plantsLayer_title": "Pflanzen-Ebene",

```
```